src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -262,9 +262,10 @@ const DataAccessRequestApplication = (props) => {
 
       // TS thinks that collection.dars is an object, but it is a map
       const darMap = new Map(Object.entries(dars));
-      setReverseOrderedDARs(
-          [...darMap.values()].sort((a, b) => b.id - a.id)
-      );
+      const reverseOrderedDARs = [...darMap.values()].sort((a, b) => b.id - a.id)
+
+
+      setReverseOrderedDARs(reverseOrderedDARs);
```

Comment:
Heads up that there's a bug at line 269 below that I've fixed in https://github.com/DataBiosphere/duos-ui/pull/2968.

The collection returned isn't ordered so we might be getting the wrong DAR at 273.